src/api/app/policies/bs_request_policy.rb

```diff
@@ -7,4 +7,8 @@ def create?
 
     false
   end
+
+  def can_handle_request?(is_target_maintainer, is_author)
```

Comment:
To follow Pundit conventions, we should rename this method to `handle_request?` and encapsulate the logic from `is_target_maintainer` and `is_author` into this method. This way, we don't have to duplicate the logic whenever calling this Pundit policy.

The method signature would become:

```suggestion
  def handle_request?
```